ranking/src/ts_system/normal.rs

```diff
@@ -0,0 +1,183 @@
+extern crate overload;
+
+use overload::overload;
+use std::ops;
+
+use statrs::function::erf::erfc;
+use std::f64::consts::PI;
+use std::f64::INFINITY;
+
+const PREC: f64 = 1e-4;
+
+#[derive(Clone, Debug)]
+pub struct Gaussian {
```

Comment:
In case it's helpful at all, the `statrs` crate that I'm using already contains Gaussians: https://docs.rs/statrs/0.13.0/statrs/distribution

Reply:
I'd rather not mess with the internals of a working TS implementation.